content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/ups_credentials.rst

```diff
@@ -1,68 +1,210 @@
-=====================================================
-How to get UPS credentials for integration with Odoo?
-=====================================================
+===============
+UPS integration
+===============
 
-In order to use the Odoo UPS API, you will need:
+UPS is a shipping carrier service that integrates with Odoo to coordinate shipping to all regions.
+Once integrated, users can create shipping methods that estimates shipping costs and :doc:`generate
+labels <../advanced_operations_shipping/labels>`.
 
-- A UPS.com user ID and password
+.. seealso::
+   :doc:`third_party_shipper`
 
-- A UPS account number
+To set up the UPS shipping connector in Odoo, complete these steps:
 
-- An Access Key
+#. Create a UPS account and get the credentials
+#. Configure company information in UPS
+#. Set up shipping method in Odoo
 
 An Access Key is a 16 character alphanumeric code that allows access to
 the UPS Developer Kit API Development and Production servers.
 
-Create a UPS Account
-====================
+UPS account setup
+=================
 
-Note that only customers located in the US can open a UPS account online.
-If you are located outside the US, you will need to contact UPS Customer
-Service in order to to open an account.
+To get started, go to the `UPS website <https://www.ups.com>`_ and click the :guilabel:`Log In`
+button in the top-right corner to log in or create a UPS account.
 
-You can read additional information about opening a UPS account on the
-their website, on the page, `How to Open a UPS Account Online
-<https://www.ups.com/content/us/en/resources/sri/openaccountonline.html?srch_pos=2&srch_phr=open+ups+account>`_
+After logging in, click the profile icon in the top-right corner, and select :guilabel:`Accounts and
+Payments` from the drop-down menu.
 
-If you don't already have a UPS account, you can create one along with
-your online profile by following these steps:
+.. image:: ups_credentials/accounts-payment.png
+   :align: center
+   :alt: Show how to navigate to the "Accounts and Payment" page from the home screen.
 
-1. Access the UPS.com web site at
-`www.ups.com <http://www.ups.com/>`__, and click the **New User** link
-at the top of the page.
+On the :guilabel:`Accounts and Payments` page, two accounts must be configured: an Odoo shipment
```

Comment:
```suggestion
On the :guilabel:`Accounts and Payment Options` page, two accounts must be configured: an Odoo shipment
```
![ups-accounts-and-payment-options](https://github.com/odoo/documentation/assets/5496809/a738b1ca-ccf8-4434-aed1-ed705556e839)